job/server/src/main/java/alluxio/job/migrate/MigrateDefinition.java

```diff
@@ -308,10 +311,12 @@ private static void migrate(MigrateCommand command, WritePType writeType, boolea
 
     CreateFilePOptions createOptions =
         CreateFilePOptions.newBuilder().setWriteType(writeType).build();
-
-    try (FileOutStream out = fileSystem.createFile(new AlluxioURI(destination), createOptions)) {
-      try (FileInStream in = fileSystem.openFile(new AlluxioURI(source))) {
-        IOUtils.copy(in, out);
+    OpenFilePOptions openFileOptions =
+        OpenFilePOptions.newBuilder().setReadType(ReadPType.NO_CACHE).build();
```

Comment:
I don't like hardcoding the read type, there could be some cases where the user wants to cache, could we pass it in as well?

Reply:
do you mean put readtype  in `MigrateConfig`and set `NO_CACHE` in `distributed{Cp,Mv}`?

Reply:
Ideally we would always use the user defined read/write mode and perhaps change the default read mode to `NO_CACHE` for cp/mv but allow the user to override that if they want.

Reply:
I am for making Migrate job more flexible , 
but I am hesitate to provide more CLI tuning flags to users at this point.

Reply:
Sure, we can change this when we run into the case where `NO_CACHE` hardcoded is a problem.